draft-ietf-anima-brski-prm.md

```diff
@@ -293,36 +277,36 @@ This presents a rendezvous problem: the pledge is unavailable for certain period
 To overcome this situation, the pledges may need to be powered on, either manually or by sending a trigger signal. 
 
 
-
 # Requirements Discussion and Mapping to Solution-Elements {#req-sol}
 
-Based on the intended target environment described in {{sup-env}} the following requirements are derived to support bootstrapping of pledges in responder mode (acting as server).
+Based on the intended target environment described in {{sup-env}}, the following requirements are derived to support bootstrapping of pledges in responder mode (acting as server):
 
-* To facilitate the communication between a pledge in responder mode and registrar, additional functionality is needed either on the registrar or as a stand-alone component.
-  This new functionality is defined as registrar-agent and acts as an agent of the registrar to trigger the pledge to generate requests for voucher and enrollment. These requests are then provided by the  registrar-agent to the registrar.
+* To facilitate the communication between a pledge in responder mode and the registrar, additional functionality is needed either on the registrar or as a stand-alone component.
+  This new functionality is defined as registrar-agent and acts as an agent of the registrar to trigger the pledge to generate requests for voucher and enrollment.
+  These requests are then provided by the registrar-agent to the registrar.
   This requires the definition of pledge endpoints to allow interaction with the registrar-agent.
 
-* The communication between the registrar-agent and the pledge MUST NOT rely on transport layer security (TLS) because the pledge does not have a certificate that can easily be verified by {{RFC6125}} methods.
-It is also more difficult to use TLS over other technology stacks (e.g., BTLE).
+* The communication between the registrar-agent and the pledge must not rely on Transport Layer Security (TLS) because the pledge does not have a certificate that can easily be verified by {{RFC6125}} methods.
```

Comment:
To my understanding this cannot be a normative statement; we are in an informative requirements section...

Reply:
agreed